modules/userId/index.js

```diff
@@ -820,6 +821,16 @@ export function init(config) {
       auctionDelay = utils.isNumber(userSync.auctionDelay) ? userSync.auctionDelay : NO_AUCTION_DELAY;
       updateSubmodules();
     }
+    // userSync.ppid should be one of the 'source' values in getUserIdsAsEids() eg pubcid.org or id5-sync.com
+    if (userSync && userSync.userIds && userSync.ppid && findIndex(getUserIdsAsEids(), x => x.source === userSync.ppid) >= 0 && typeof getUserIdsAsEids().find(x => x.source === userSync.ppid).uids[0].id === 'string') {
+      window.googletag = window.googletag || { cmd: [] };
+      const ppid = getUserIdsAsEids().find(x => x.source === userSync.ppid).uids[0].id.replace(/[\W_]/g, '');
+      if (ppid.length >= 32 && ppid.length <= 150) {
```

Comment:
is this check due to ppid requirements from google?

Reply:
yes

Reply:
https://github.com/prebid/prebid.github.io/pull/2788 is the docs pr btw